### PR TITLE
feat(WEBRTC-53): migration verto to telnyxRTC

### PR DESCRIPTION
**Pull request by DeividVeloso**
- Add support to modules verto and telnyx_rtc
 - Rename public interfaces from Verto to TelnyxRTC

## 📝 To Do

- [x] All linters pass
- [x] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. Access http://localhost:6006/?path=/story/webdialer--example
2. Try to make a call

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/TelnyxRTCClient/TelnyxRTCClient.test.ts [hunk 1]
```diff
@@ -23,8 +23,8 @@ Object.defineProperty((global as any).navigator, 'mediaDevices', {
   configurable: true,
 });
 
-describe('VertoClient', () => {
-  const client = new VertoClient({
+describe('TelnyxRTCClient', () => {
```
Comment (SuaYoo) [inline src/TelnyxRTCClient/TelnyxRTCClient.test.ts]: Could you add a spec for the new `module` parameter?

File: src/TelnyxRTC/Verto.ts [hunk 2]
```diff
@@ -130,6 +132,7 @@ export default class Verto {
     this.dialog = null;
     this.dialogs = {};
     this.params = params;
+    this.module = params.module;
```
Comment (SuaYoo) [inline src/TelnyxRTC/Verto.ts]: Can you throw an error early here if `module` is not `verto` or `telnyx_rtc`? That way we don't force the caller to wait until a method is called to throw a type error.

Comment (DeividVeloso) [reply src/TelnyxRTC/Verto.ts]: I will do that

File: src/TelnyxRTCClient/TelnyxRTCClient.test.ts (hunk 1 above)
Comment (DeividVeloso) [reply src/TelnyxRTCClient/TelnyxRTCClient.test.ts]: Done!

**Pull request merged**